tests/integration/test_integration.py

```diff
@@ -1748,6 +1748,43 @@ def test_copy(client: TestClient, admin_access_token: str, internal_study_id: st
     assert res["public_mode"] == "READ"
 
 
+def test_copy_variant_as_raw(client: TestClient, admin_access_token: str, internal_study_id: str) -> None:
+    client.headers = {"Authorization": f"Bearer {admin_access_token}"}
+
+    raw = client.post("/v1/studies?name=raw")
+    assert raw.status_code == 201
+    parent_id = raw.json()
+    client.post(
+        f"/v1/studies/{parent_id}/areas",
+        json={"name": "area1", "type": "AREA"},
+    )
+    client.post(
+        f"/v1/studies/{parent_id}/areas",
+        json={"name": "area2", "type": "AREA"},
+    )
+
+    var = client.post(f"/v1/studies/{parent_id}/variants", params={"name": "variant"})
+    assert var.status_code == 200
+    variant_id = var.json()
+
+    variant_study = client.get(f"/v1/studies/{variant_id}")
+    assert variant_study.status_code == 200
+
+    copy = client.post(f"/v1/studies/{variant_id}/copy?dest=copied&use_task=false")
+    client.get(f"/v1/tasks/{copy.json()}?wait_for_completion=True")
+
+    all_studies = client.get("/v1/studies")
+    assert variant_study.status_code == 200
+    assert len(all_studies.json()) == 4
+
+    copied_study = client.get("/v1/studies?name=copied")
+    assert copied_study.status_code == 200
+    copied_id = next(iter(copied_study.json()))
+
+    copied_areas = client.get(f"/v1/studies/{copied_id}/areas")
+    assert copied_areas.json() == client.get(f"/v1/studies/{parent_id}/areas").json()
+
```

Comment:
We should add a test for the copy of output, since we have seen that it's not that obvious to make it work :)